common/persistence/src/main/java/app/coronawarn/server/common/persistence/repository/TestDataUploadRepository.java

```diff
@@ -0,0 +1,33 @@
+package app.coronawarn.server.common.persistence.repository;
+
+import app.coronawarn.server.common.persistence.domain.DiagnosisKey;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.jdbc.repository.query.Modifying;
+import org.springframework.data.jdbc.repository.query.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Profile("testdata")
+public interface TestDataUploadRepository
```

Comment:
If possibile this should be moved to the context which actually uses it, such that other modules which depend on it don't have the visibility.  The only thing to add is an extra component scan to cover the new package